jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -278,9 +270,33 @@ rvsdg::node *
 MlirToJlmConverter::ConvertOperation(
     ::mlir::Operation & mlirOperation,
     rvsdg::region & rvsdgRegion,
-    std::vector<rvsdg::output *> & inputs)
+    ::llvm::SmallVector<rvsdg::output *> & inputs)
 {
-  if (::mlir::isa<::mlir::rvsdg::OmegaNode>(&mlirOperation))
+
+  /* #region Arithmetic Integer Operation*/
+  //! Here the LLVM dialect where only implemented for AddOp. Other operation should maybe be
+  //! imported Need to choose which one of mlir::arith or mlir::LLVM to use for the MLIR
+  //! representation
+  rvsdg::node * convertedNode = ConvertBitBinaryNode(mlirOperation, inputs);
+  // If the operation was converted it means it has been casted to a bit binary operation
+  if (convertedNode)
+    return convertedNode;
```

Comment:
So you use the check for two inputs whether it is a bit binary or not? That sounds a little bit off.

Reply:
See comment above.